addons/website_mass_mailing/views/mailing_templates_portal_unsubscribe.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="unsubscribe_form" inherit_id="mass_mailing.unsubscribe_form">
+        <xpath expr="//div[hasclass('o_container_small')]" position="inside">
+            <div class="mb-5 py-4 w-100 text-start oe_structure">
+                <div class="container">
+                    <h3>Not a fan of emails ?</h3>
+                    <p>Follow us on social Media</p>
+                    <div class="s_social_media o_not_editable text-start" data-snippet="s_social_media" data-name="Social Media">
+                        <h4 class="s_social_media_title d-none o_default_snippet_text">Social Media</h4>
+                        <a href="/website/social/facebook" class="s_social_media_facebook" target="_blank">
+                            <i class="fa fa-facebook rounded shadow-sm o_editable_media"></i>
+                        </a>
+                        <a href="/website/social/twitter" class="s_social_media_twitter" target="_blank">
+                            <i class="fa fa-twitter rounded shadow-sm o_editable_media"></i>
+                        </a>
```

Comment:
There is an ongoing PR changing the Twitter logo to the `X` one, so be aware that you may need to change this icon if the snippet changed names.

Reply:
TODO https://github.com/odoo/odoo/pull/159360